### PR TITLE
Timeout in idle

### PR DESCRIPTION
**Pull request by dstrigoun**

### CODE REVIEW

File: Final_Project/Final_Project.vcxproj.filters [hunk 1]
```diff
@@ -42,7 +42,11 @@
       <Filter>Header Files</Filter>
     </ClInclude>
     <ClInclude Include="Main.h">
+<<<<<<< HEAD
```
Comment (jmkim2392) [inline Final_Project/Final_Project.vcxproj.filters]: there's a merge conflict, I had created similar directories in the Header directory for app, dl, and phys
We can keep those or just simply have all in one Header directory.

File: Final_Project/Main.cpp [hunk 2]
```diff
@@ -186,4 +203,72 @@ int randomNumberGenerator(int min, int max)
...
+--	NOTES:
+--	Called by the thread created to periodically check the difference between
+--  the current time and the time when the last EOT was received
+--------------------------------------------------------------------------------------*/
+DWORD WINAPI checkIdleTimeout(LPVOID n)
+{
+	while (1) {
+		time_t currentTime = time(0);
+
+		char cur[16] = "";
```
Comment (jmkim2392) [inline Final_Project/Main.cpp]: These are just for debugging purposes right?

Comment (jmkim2392) [reply Final_Project/Main.cpp]: Everything else looks good and works great! Good job!

File: Final_Project/Final_Project.vcxproj.filters (hunk 1 above)
Comment (dstrigoun) [reply Final_Project/Final_Project.vcxproj.filters]: Ah I totally missed that! I think that's from the temp `Main.h` file I had committed. I'll make it so we keep the separate directories in the Header directory.

**Review by jmkim2392 — APPROVED** [review]
Looks good to me!

**Pull request merged**